src/main/java/frc/robot/Constants.java

```diff
@@ -66,11 +70,11 @@ public class PathPlanner {
 
         public static final double DRIVEBASE_RADIUS = Units.inchesToMeters(14.5); // meters
 
-        public static final double TRANSLATION_P = 10;
+        public static final double TRANSLATION_P = 8.25;
         public static final double TRANSLATION_I = 0;
         public static final double TRANSLATION_D = 0;
 
-        public static final double ROTATION_P = 1;
+        public static final double ROTATION_P = 0.000;
```

Comment:
on purpose?

Reply:
Yes, also don't see us using path planner the rest of this comp anyways